source/js/buyers-guide/analytics-events.js

```diff
@@ -225,17 +225,21 @@ const ProductGA = {
     // bind GA events that have special conditions
 
     // tracks when MSS accordion on product page is expanded
-    bindCheckboxCheckedGA("#view-product-page #mss-accordion-toggle", {
-      category: `product`,
-      action: `security expand accordion tap`,
-      label: `detail view for MSS on for ${productName}`,
-    });
+    if (document.querySelector(`#view-product-page`)) {
+      bindCheckboxCheckedGA("#view-product-page #mss-accordion-toggle", {
+        category: `product`,
+        action: `security expand accordion tap`,
+        label: `detail view for MSS on for ${productName}`,
+      });
+    }
 
-    bindCheckboxCheckedGA("body.catalog #product-filter-pni-toggle", {
-      category: `buyersguide`,
-      action: `ding checkbox checked`,
-      label: `ding checkbox checked on ${pageTitle}`,
-    });
+    if (document.querySelector(`body.catalog`)) {
```

Comment:
same thing here - prevents trying to bind GA on PNI pages that are guaranteed to not work because they're not a catalog page (homepage/category)